### ISSUE TITLE
test_keyword_match failing

### ISSUE THREAD

**Issue by jluttine** [tholo/pytest-flake8#44]
Test `test_keyword_match` is failing on my computer. Here's the log: [pytest-flake8.log](https://github.com/tholo/pytest-flake8/files/1700509/pytest-flake8.log)

Here's the output of `pip freeze`:

```
attrs==17.4.0
flake8==3.5.0
mccabe==0.6.1
pluggy==0.6.0
py==1.5.2
pycodestyle==2.3.1
pyflakes==1.6.0
pytest==3.3.2
pytest-flake8==0.9.1
six==1.11.0
```

What could be wrong?

**Comment by jluttine** [comment]
I'm packaging this for a Linux distro and recently the package broke, I suppose because of pytest upgrade. It would be nice to get this fixed. Is there any workaround or can I safely ignore the failing test?

**Comment by tholo** [comment]
This had already been fixed, but not published -- should be fixed in 1.0.0 that is being released right now.
